packages/relay/src/lib/eth.ts

```diff
@@ -1614,7 +1644,7 @@ export class EthImpl implements Eth {
     if (blockNumberOrTag != null && blockNumberOrTag.length < 32 && !isNaN(blockNumber)) {
       const latestBlockResponse = await this.mirrorNodeClient.getLatestBlock(requestId);
       const latestBlock = latestBlockResponse.blocks[0];
-      if (Number(blockNumberOrTag) > latestBlock.number + maxBlockRange) {
+      if (Number(blockNumberOrTag) > latestBlock.number + this.maxBlockRange) {
```

Comment:
Why not use the identical `blockNumber` instead of `Number(blockNumberOrTag)` here?

Reply:
Updated